cooked-validators/src/Cooked/MockChain/UtxoState/Testing.hs

```diff
@@ -31,7 +31,7 @@ equalModuloAda a b = testAll (eqModAda . snd) $ M.toList $ utxoStateDiff a b
     hasOnlyAda v = case Pl.flattenValue v of
       [] -> testSuccess
       [(sym, tok, _)] -> sym .==. Ada.adaSymbol .&&. tok .==. Ada.adaToken
-      _ -> testFailureMsg "Value has more than just Ada"
+      _ -> testFailureMsg $ "Value has more than just Ada: " ++ show v
```

Comment:
Weird to have 2 very similar `hasOnlyAda` functions.

Reply:
What is the second one?

Reply:
What is the other one?

Reply:
The last function declared in MockChain/UtxoPredicate.hs.

Reply:
Lol! Found it; it was right there. Good catch! This was a copy-paste oversight, there should be only one such function.